manywheel/build_rocm.sh

```diff
@@ -333,8 +369,14 @@ echo "PYTORCH_ROCM_ARCH: ${PYTORCH_ROCM_ARCH}"
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 if [[ -z "$BUILD_PYTHONLESS" ]]; then
-    BUILD_SCRIPT=build_common.sh
-else
-    BUILD_SCRIPT=build_libtorch.sh
+    BUILD_SCRIPT=build_torch_wheel.sh	
 fi
 source $SCRIPTPATH/${BUILD_SCRIPT}
+if [[ "$BUILD_LIGHTWEIGHT" == "1" ]]; then
+    do_lightweight_build
+fi
+
+if [[ "$BUILD_HEAVYWEIGHT" == "1" ]]; then
+    do_heavyweight_build
+fi
+
```

Comment:
We will probably want to `rm -rf /tmp/wheelhouserocm6.4.0` here so that both lightweight and heavyweight functions can access the wheel.